views/courses/components/topics.hbs

```diff
@@ -1,6 +1,11 @@
 <div class="row">
 	{{#userHasPermission "COURSE_EDIT"}}{{#unless @root.isArchived}}
 			<div class="col-sm-12">
+                <a href="#" data-courseId="{{../_id}}" class="btn btn-secondary btn-sm pull-right ml-1 btn-import-topic-from-template"
+                   title="Thema aus Vorlage">
+                    <i class="fa fa-exchange" aria-hidden="true"></i>
```

Comment:
```suggestion
                    <i class="fa fa-flask" aria-hidden="true"></i>
```

![image](https://user-images.githubusercontent.com/22987140/69056991-5e3d0300-0a11-11ea-8f97-2c6c2920084f.png)

https://fontawesome.com/v4.7.0/icon/flask

Reply:
thx, I like this icon 👍

Reply:
like it too